src/server/lib/idapi/IDAPICookies.ts

```diff
@@ -37,9 +37,22 @@ export const setIDAPICookies = (
       }
     })
     .forEach(({ key, value, sessionCookie = false }) => {
+      const expires: Date | undefined = (() => {
+        if (key === 'SC_GU_LA') {
+          // have SC_GU_LA cookie expire in 30 mins, despite the fact that it is a session cookie
```

Comment:
If this behaviour is to do with differences in browser handling, would it be possible to check if this is being called with an `appClientId` and only update the behaviour in that case? Since it appears to work predictably in regular browsers?

Reply:
I don't think we need to, this solution covers all browsers and behaviours, it doesn't need to be a session cookie in normal browsers either.

Reply:
We'd be adding complexity for something that doesn't make any difference.

Reply:
Oh, great then!